src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -83,12 +82,31 @@ public TileEntity createNewTileEntity(World world, int meta)
         return new TileEntityColonyBuilding();
     }
 
+    /**
+     * Convert the given metadata into a BlockState for this Block
+     */
+    @NotNull
     @Override
-    public int getMetaFromState(@NotNull IBlockState state)
+    public IBlockState getStateFromMeta(int meta)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)